optimum/bettertransformer/models/__init__.py

```diff
@@ -97,6 +98,10 @@ class BetterTransformerManager:
         },
         "markuplm": {"MarkupLMLayer": BertLayerBetterTransformer},
         "mbart": {"MBartEncoderLayer": MBartEncoderLayerBetterTransformer},
+        "musicgen": {
+            "T5Attention": T5AttentionLayerBetterTransformer,
+            "MusicgenAttention": MusicGenAttentionLayerBetterTransformer,
```

Comment:
Is it sufficient just to transform the attention layer @fxmarty, or do we need to convert the whole transformer block, as per this guide? https://huggingface.co/docs/optimum/bettertransformer/tutorials/contribute#step-2-building-the-xxxlayerbettertransformer-module

Reply:
No it is good as is!